source/integration.rst

```diff
@@ -1,7 +1,20 @@
 Integration
 -----------
 
-You can follow dedicated integration guides for each cloud management framework:
+Integration of cloud stacks into EGI FedCloud follows a well-defined path, with certain steps which need to be taken, depending on the cloud stack in question. By integration here, we refer to the proper interoperation with EGI infrastructure services such as accounting, monitoring, authentication and authorisation, _etc_. These configurationsmake your site discoverable and usable by the communities you wish to support, and allow EGI to support you in operational and technical matters.
```

Comment:
Missing space

Reply:
I can live with one sentence per line, thanks for the suggestion.